themes/startwords/assets/scss/article/_summary.scss

```diff
@@ -149,3 +149,33 @@ $preview-height: rem(250px);
         }
     }
 }
+
+
+.other-articles {
+    clear: both;
+    padding-top: 2rem;
```

Comment:
i tried adding another short piece and everything looked good but there wasn't much spacing in between them on the issue list page - might want a margin-bottom here, or alternatively a rule like
```scss
.article-short-summary + .article-short-summary {
    margin-top: rem(10px);
}
```